examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -56,6 +56,7 @@ model:
   hidden_size: 768
   ffn_hidden_size: 3072 # Transformer FFN hidden size. Usually 4 * hidden_size.
   num_attention_heads: 12
+  num_tokentypes: 2
```

Comment:
Can you add a documentation saying its 0 when not doing Next sentance prediction, 2 when doing bert  binary head , and 1 just to be compatible with Hugginf face.